pkg/store/bucket.go

```diff
@@ -2625,6 +2625,13 @@ func (pg postingGroup) mergeKeys(other *postingGroup) *postingGroup {
 	return &pg
 }
 
+func min(a, b int) int {
```

Comment:
I think we have these functions in go 1.21, so maybe we can just use them?

Reply:
Good point. I will take a look at how to do it.

Reply:
I think it's fine to do it, eventually we'll want to use features from new Go versions.

Reply:
@fpetkovski Updated! PTAL